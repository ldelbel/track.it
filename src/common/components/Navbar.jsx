import React from 'react';
import styles from '../styles/Navbar.module.scss';

const Navbar = () => (
  <nav className={styles.container}>
    <h1>Track.it</h1>
  </nav>
);

export default Navbar;
