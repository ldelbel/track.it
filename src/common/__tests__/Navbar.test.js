import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

describe('navbar component', () => {
  it('renders correctly', () => {
    const navbar = renderer.create(<Navbar />).toJSON();
    expect(navbar).toMatchSnapshot();
  });
});
