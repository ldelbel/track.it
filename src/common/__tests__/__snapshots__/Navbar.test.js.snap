// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar component renders correctly 1`] = `
<nav
  className="container"
>
  <h1>
    Track.it
  </h1>
</nav>
`;
